python-test/features/steps/control_plane_policies.py

```diff
@@ -192,27 +159,79 @@ def delete_policy(token, policy_id):
                 + policy_id + ' failed with status=' + str(response.status_code))
 
 
-def check_logs_contain_message_for_policies(logs, expected_message, policies_list,
-                                          timestamp_considered=datetime.now().timestamp()):
+def check_logs_contain_message_for_policies(logs, expected_message, list_agent_policies_id, considered_timestamp):
     """
-    Checks agent container logs for expected message for all applied policies
+    Checks agent container logs for expected message for all applied policies and the log analysis loop is interrupted
+    as soon as a log is found with the expected message for each applied policy.
 
     :param (list) logs: list of log lines
     :param (str) expected_message: message that we expect to find in the logs
-    :param (str) policies_list: list with all policy id applied to the agent
-    :param timestamp_considered: timestamp from which the log will be considered.
-                                                                Default: timestamp at which behave execution is started
-    :returns: (bool) whether expected message was found in the logs for expected policy
+    :param (list) list_agent_policies_id: list with all policy id applied to the agent
+    :param (float) considered_timestamp: timestamp from which the log will be considered
```

Comment:
Sorry to only catch this now, but timestamp should be an integer, no?

Reply:
No. Timestamp is a float. 
https://docs.python.org/3/library/datetime.html

` datetime.timestamp()
Return POSIX timestamp corresponding to the datetime instance. The return value is a float similar to that returned by time.time().`